src/commands/helm-ct-lint.yaml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
I'd remove `-ct` part from the name. I think tooling we use here is irrelevant. That way in the future we can change the tooling and not change the name remaining backward compatible.